.github/workflows/extract-translations.yml

```diff
@@ -0,0 +1,41 @@
+on:
+  push:
+    branches:
+      - dev
+
+jobs:
+  sync-translations:
+    name: Extract translations and submit a PR
+    runs-on: ubuntu-latest
+    env:
+      BRANCH: ${{ github.ref_name }}
+    steps:
+      - name: Checkout module-translation-tool
+        uses: actions/checkout@v4
+        with:
+          repository: PrestaShopCorp/module-translation-tool
+          ref: main
+      - name: Create config for module-translation-tool
+        run: |
+          cp ./module.cfg.example ./module.cfg
+          sed -i 's/^MODULE_NAME.*/MODULE_NAME="autoupgrade"/g' ./module.cfg
+          sed -i 's/^GIT_REPO_USERNAME.*/GIT_REPO_USERNAME="PrestaShop"/g' ./module.cfg
+          sed -i 's/^GIT_REPO_NAME.*/GIT_REPO_NAME="autoupgrade"/g' ./module.cfg
```

Comment:
Could be interesting to rely on event contents to make this workflow generic, instead of having to specify the name and organization everytime we copy this file in a new repository.

Reply:
Yes good idea ! It's done ! Thank you for you review ;)